src/event_handler.py

```diff
@@ -50,6 +50,12 @@ def store_queued_events(_, __):
         try:
             decrypted_message = decrypt_message(message['Body'], decryption_key)
             event = event_from_json(decrypted_message)
+
+            # Send audit events to this lambda function's CloudWatch log group.
+            # This is the raw JSON event on a line by its self so Splunk can
+            # parse it as JSON.
+            print(decrypted_message)
```

Comment:
Is there a reason we used `print()` rather than `logger.info()`?

Reply:
The noise from the logger breaks the automatic parsing of the JSON event in Splunk. It can still be done but requires extracting the JSON object from the line, parsing it, then running another search over that data.